src/components/trade/PriceImpact/utils.ts

```diff
@@ -0,0 +1,117 @@
+import BigNumber from 'bignumber.js'
+
+import { parseBigNumber } from 'utils'
+import { ONE_HUNDRED_BIG_NUMBER, ONE_BIG_NUMBER, ZERO_BIG_NUMBER } from 'const'
+
+import { PriceImpactArgs, PriceImpactArgsBase } from '../PriceImpact'
+
+// Limit as to where price colour changes
+const HIGH_PRICE_IMPACT_THRESHOLD = new BigNumber(5) // 5%
+const MID_PRICE_IMPACT_THRESHOLD = new BigNumber(3) // 3%
+const LOW_PRICE_IMPACT_THRESHOLD = ONE_BIG_NUMBER // 1%
+// TODO: remove for slippage toggle
+// 0.5% placeholder slippage
+const HIGH_PLACEHOLDER_SLIPPAGE = new BigNumber(1.05) // 5%
+const MID_PLACEHOLDER_SLIPPAGE = new BigNumber(1.03) // 3%
+
+enum ImpactLevel {
```

Comment:
nice! Although, a nitipick is that I think u don't need to specify the numbers, since we work only with the enum, right. They are self assigned

Reply:
done in #1550